docs/howto/custom-template-tags.txt

```diff
@@ -498,6 +498,127 @@ you see fit:
     {% current_time "%Y-%m-%d %I:%M %p" as the_time %}
     <p>The time is {{ the_time }}.</p>
 
+.. _howto-custom-template-tags-simple-block-tags:
+
+Simple block tags
+-----------------
+
+.. method:: django.template.Library.simple_block_tag()
+
+Passing arguments may not be enough for some usages -- you may need to pass chunks of
+rendered template into your custom tags. This can be achieved using the
+``simple_block_tag`` helper function of ``django.template.Library``.
+``simple_block_tag`` functions similar to :meth:`~django.template.Library.simple_tag()`,
+however additionally passes a ``content`` argument which contains its rendered template
+content.
+
+For example, a custom block tag which created a chart could look like this::
+
+    from django import template
+    from myapp.charts import render_chart
+
+    register = template.Library()
+
+
+    @register.simple_block_tag
+    def chart(content):
+        return render_chart(source=content)
+
+The ``content`` argument contains everything in between the ``{% chart %}`` and
+``{% endchart %}`` tags. If there are other template tags or variables, they will be
+rendered before being passed in.
+
+.. code-block:: html+django
+
+    {% chart %}
+        digraph G {
+            label = "Chart for {{ request.user }}"
+            A -> {B C}
+        }
+    {% endchart %}
+
+A few things to note about the ``simple_block_tag`` helper function:
+
+* The first argument must be called ``content``, and will contain the contents of the
+  template tag as a rendered string.
+* Checking for the required number of arguments, etc., has already been
+  done by the time our function is called, so we don't need to do that.
+* The quotes around the argument (if any) have already been stripped away,
+  so we receive a plain string.
+* If the argument was a template variable, our function is passed the
+  current value of the variable, not the variable itself.
```

Comment:
```suggestion
* If an argument was a template variable, our function is passed the current
  value of the variable, not the variable itself.
```
As we already have the argument "content", all other arguments are essentially additional and optional I think